src/main/java/frc/robot/commands/sequences/BaseAutonSequence.java

```diff
@@ -26,194 +24,23 @@ public abstract class BaseAutonSequence extends SequentialCommandGroup {
-        )),
-        X7(new Pose2d(
-            Units.inchesToMeters(BLUE_PLACEX),
-            Units.inchesToMeters(64.818),
-            Rotation2d.fromDegrees(180)
-        )),
-        X8(new Pose2d(
-            Units.inchesToMeters(BLUE_PLACEX),
-            Units.inchesToMeters(41.761),
-            Rotation2d.fromDegrees(180)
-        )),
-        X9(new Pose2d(
-            Units.inchesToMeters(BLUE_PLACEX),
-            Units.inchesToMeters(20.016),
-            Rotation2d.fromDegrees(180)
-        )),
-
-        PIECE1(new Pose2d(
-            Units.inchesToMeters(0),
-            Units.inchesToMeters(0),
-            Rotation2d.fromDegrees(0)
-        )),
-        PIECE2(new Pose2d(
-            Units.inchesToMeters(0),
-            Units.inchesToMeters(0),
-            Rotation2d.fromDegrees(0)
-        )),
-        PIECE3(new Pose2d(
-            Units.inchesToMeters(0),
-            Units.inchesToMeters(0),
-            Rotation2d.fromDegrees(0)
-        )),
-        PIECE4(new Pose2d(
-            Units.inchesToMeters(0),
-            Units.inchesToMeters(0),
-            Rotation2d.fromDegrees(0)
-        )),
-        BALANCE_INIT(new Pose2d(
-            Units.inchesToMeters(BLUE_INITX),
-            Units.inchesToMeters(107.638),
-            Rotation2d.fromDegrees(180)
-        ));
-
-        public Pose2d position;
-
-        private BluePositions(Pose2d position){
-            this.position = position;
-        }
-    }
-
     /**
-     * non balancing TOP auton sequence
-     * @param robotContainer
-     * @param initialPose initPose
-     * @param midPose1 midPose1
-     * @param midPose2 midPose2
-     * @param midpose3 misPose3
-     * @param placePose placepose of first gamepeice
-     * @param grabPose grabpose of 2nd gamepiece
-     * @param placePose2 placepose of 2nd gamepiece
-     * @param elevatorPosition The height to place the 1st game piece.
-     * @param elevatorPosition2 The height to place the 2nd game piece.
+     * Non-balancing top auton sequence.
+     * @param swerveSubsystem The swerve subsystem.
+     * @param rollerSubsystem The roller subsystem.
+     * @param tiltedElevatorSubsystem The tilted elevator subsystem.
+     * @param initialPose The initial pose of the robot.
+     * @param midPose1 The first midpose of the sequence.
+     * @param midPose2 The second midpose of the sequence.
+     * @param midpose3 The third midpose of the sequence.
+     * @param placeState The state of the robot when placing the first game piece (pose and elevator state).
+     * @param grabPose The pose to grab the second game piece at.
+     * @param placeState2 The state of the robot when placing the second game piece (pose and elevator state).
      */
     public BaseAutonSequence(
         BaseSwerveSubsystem swerveSubsystem, RollerSubsystem rollerSubsystem, TiltedElevatorSubsystem tiltedElevatorSubsystem,
         Pose2d initialPose, Pose2d midPose1, Pose2d midPose2, Pose2d midpose3, 
```

Comment:
it would be nice if these midPose args were more descriptive: enter/exit community?